### DISCUSSION TITLE
Disable Speed Insights Speed Insights tracking and storage for your project will be disabled immediately.

### DISCUSSION THREAD

**Discussion by IVEN21** [vercel/community#6197] [Feature Question] [answered]
### Question

I have a existing blob storage, if disabling Speed Insights, does it disabled my storage on blob also? As it said my storage will be also disabled, I'm not sure what the storage refer to

![image](https://github.com/vercel/community/assets/66749048/2dbd6ebe-f014-4abc-aec5-a59cb0539c4d)

**Comment by swarnava** [comment, marked as answer]
Hey there,

Disable Speed Insights should only disable "Storing" of Speed Insights data. Your blob storage will unaffected. :)

**Reply by IVEN21** [reply]
Thanks, I would say the wording was misleading, if it can be improved, it would benefit future users.

**Reply by swarnava** [reply]
Indeed this is confusing. We have an internal ticket to rephrase this and we will ship this shortly. Thanks for reporting. :)
